service/src/main/resources/db/migration/V219__message_account_evaka_user_id.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE message_account ADD COLUMN evaka_user_id uuid UNIQUE REFERENCES evaka_user (id);
+UPDATE message_account SET evaka_user_id = coalesce(person_id, employee_id);
+UPDATE message_account SET employee_id = NULL WHERE employee_id IS NOT NULL AND NOT active;
+ALTER TABLE message_account DROP COLUMN active;
+
+ALTER TABLE message_account
+    DROP CONSTRAINT message_account_person_id_fkey,
+    ADD CONSTRAINT message_account_person_id_fkey FOREIGN KEY (person_id) REFERENCES person (id) ON DELETE SET NULL,
+    DROP CONSTRAINT message_account_created_for_fk,
+    ADD CONSTRAINT message_account_owner_fk CHECK (
```

Comment:
It just crossed my mind that foreign key naming convention is `fk$person`. I don't think we have a convention for check constraints (just one occurrence of `chk$valid_role`).